hobbit (4.2.0.dfsg-13) unstable; urgency=low

  * Do not ignore tmpfs mountpoints in hobbitclient-linux.sh, but add
    /dev, /dev/shm, and /lib/init/rw to the default IGNORE list in
    /etc/hobbit/hobbit-clients.cfg. (Closes: #487157)

 -- Christoph Berg <myon@debian.org>  Sat, 05 Jul 2008 14:43:23 +0200

--- a/client/hobbitclient-linux.sh
+++ b/client/hobbitclient-linux.sh
@@ -42,7 +42,7 @@ uptime
 echo "[who]"
 who
 echo "[df]"
-EXCLUDES=`cat /proc/filesystems | grep nodev | awk '{print $2}' | xargs echo | sed -e 's! ! -x !g'`
+EXCLUDES=`cat /proc/filesystems | grep nodev | grep -v tmpfs | awk '{print $2}' | xargs echo | sed -e 's! ! -x !g'`
 df -Pl -x iso9660 -x $EXCLUDES | sed -e '/^[^ 	][^ 	]*$/{
 N
 s/[ 	]*\n[ 	]*/ /
--- a/hobbitd/etcfiles/hobbit-clients.cfg
+++ b/hobbitd/etcfiles/hobbit-clients.cfg
@@ -347,6 +347,10 @@
 # be placed at the end of the file.
 
 DEFAULT
+	# Ignore some usually uninteresting tmpfs mounts.
+	DISK    /dev IGNORE
+	DISK    /dev/shm IGNORE
+	DISK    /lib/init/rw IGNORE
 	# These are the built-in defaults.
 	UP      1h
 	LOAD    5.0 10.0
